tests/x11/smt.pm

```diff
@@ -54,10 +66,12 @@ sub run {
     assert_screen 'smt-installation-overview';
     send_key 'alt-n';
     wait_serial("yast2-smt-wizard-0", 400) || die 'smt wizard failed';
+}
 
-    assert_script_run 'smt-sync', 600;
-    assert_script_run 'smt-repos';
-    type_string "killall xterm\n";
+sub smt_mirror_repo {
+
+    assert_script_run 'smt-repos --enable-mirror SLES12-SP3-Installer-Updates sle-12-x86_64';
```

Comment:
I'm concerned about that hardcoded string. Either use a test variable or use existing test variable settings to combine these strings or comment explicitly why it's ok to hardcode one version/distri/arch here

Reply:
I have add comment for it, but it was hidden after I extracted this mirror function.

Smt pattern is only avaliable on x86_64 in SLE12 that means this test could be executed only on x86_64. And this case is focus on testing the basic function of SMT pattern, so any released repo on SCC is ok for it. I suppose a tiny and stable repo would be better for this case.

RMT will replace SMT in SLES15, that means this case in only be executed with SLES12SP3 and SLES12SP4 in the future. I think it is ok to hardcdoed this string as SLES12-SP3-Installer-Updates repo here.

Reply:
sure, just add a comment describing that and I am fine with the solution :)